website/docs/schema-generator/customizing-schemas/custom-type-reference.md

```diff
@@ -0,0 +1,79 @@
+---
+id: custom-type-reference
+title: Custom Types
+---
+
+Code-first has many advantages and removes duplication. However, one downside is that the types defined have to match
```

Comment:
can you elaborate more on the target use case? i.e. from the example below it is unclear why you would ever need to do it? I think that a line or two specifying that the `Foo`/`Bar` objects are defined in some dependent library JAR (so you cannot modify them in the app) would clarify the intent